gt/main.c

```diff
@@ -1770,3 +1819,90 @@ run_gt(struct net_config *net_conf, struct gt_config *gt_conf)
 	rte_atomic32_init(&gt_conf->ref_cnt);
 	return 0;
 }
+
+static lua_State *
+alloc_and_setup_lua_state(void)
```

Comment:
GT instances must use this function to create their Lua states during boot-time as well to guarantee consistency between boot-time and runtime creation of Lua states to run policies.